config/linux-x230.config

```diff
@@ -99,6 +99,11 @@ CONFIG_BLK_DEV_DM=y
 CONFIG_DM_CRYPT=y
 CONFIG_DM_VERITY=y
 CONFIG_DM_VERITY_FEC=y
+CONFIG_BLK_DEV_MD=y
```

Comment:
Those should all be modules, and those modules should be added in modules/linux section at the end, taking e1000e as an example.

Makefile should take those modules and add them in initrd if the board config keyword is added.

And init, load those modules as early as needed if the madm binary is found on to system, loading the modules in the proper order.

Otherwise, the kernel size just grew and it will fail for smaller spi boards like xx20 here.

I am not in front of computer right now, but if all those can be built as modules and not included in kernel for all (linux-x230.config being shared for all boards not being flash) this fails because if space constraints.

Let me know if you need more information!

Reply:
https://github.com/osresearch/heads/blob/9016ebccc22eabd40245743886488cf6b89b4309/modules/linux#L94

USB controller modules might be a better example to follow: https://github.com/osresearch/heads/blob/9016ebccc22eabd40245743886488cf6b89b4309/modules/linux#L94